app/code/community/Vindi/Subscription/Helper/Connector.php

```diff
@@ -133,41 +135,51 @@ private function request($endpoint, $data = [], $method = 'POST')
      *
      * @return array
      */
-    private function encrypt ($body, $endpoint)
+    public function encrypt($body, $endpoint)
     {
-        if ('payment_profile' === $endpoint) {
-            $dataToLog = $body;
+        $dataToLog = $body;
```

Comment:
Achoooo que essa atribuição não é necessária

Reply:
Está funcionando como um _setter_ 'default'.
Se remover-mos temos que alterar o return para `return $dataToLog ? : $body;`

Reply:
Tudo bem deixar o default, mas não aconselho deixar `$body`. Se por algum motivo não entrar no `if`, isso vai ficar com os dados sensíveis de cartão de crédito, certo?

Reply:
Acredito que não, no momento o único _endpont_ sensível que temos é o de `payment_profiles`.
Talvez essa preocupação faça sentido quando for incorporada a função de vínculo entre `payment_profile` e `subscription`.
Ai no caso é só adicionar o endpoint no `encrypt`.
No momento estamos enviando apenas IDs :smiley: